easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-5.3.0-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
```

Comment:
@migueldiascosta we have an easyblock dedicated to QuantumESPRESSO, why not use that?